cmd/auto-pause/auto-pause.go

```diff
@@ -121,3 +125,17 @@ func runUnpause() {
 
 	out.Step(style.Unpause, "Unpaused {{.count}} containers", out.V{"count": len(uids)})
 }
+
+func CheckIfPaused() {
```

Comment:
keep func private, if not used outside,  how about a better name ?
alreadyPaused() ?